.travis.yml

```diff
@@ -38,6 +38,6 @@ before_cache:
   - find $HOME/.ivy2/cache -name "ivydata-*.properties" | xargs rm
 cache:
   directories:
-  - $HOME/.ivy2/cache
-  - $HOME/.sbt/boot
-  - $HOME/.sbt/launchers
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot
+    - $HOME/.sbt/launchers
```

Comment:
This change is unrelated. Please revert it and amend your commit.

Reply:
Sorry, I have a linter running on save.
✅Done